packages/medusa/src/api-v2/utils/common-validators/common.ts

```diff
@@ -24,3 +25,8 @@ export const BigNumberInput = z.union([
     precision: z.number(),
   }),
 ])
+
+export const OptionalBooleanValidator = z.preprocess(
```

Comment:
Nice 👍 Let's maybe update all references to it already?

Reply:
This file already has too many changes, will do it as a follow up 👍🏻